core/src/main/scala/kafka/server/AbstractFetcherThread.scala

```diff
@@ -531,7 +548,7 @@ abstract class AbstractFetcherThread(name: String,
       true
     } catch {
       case _: FencedLeaderEpochException =>
-        onPartitionFenced(topicPartition)
+        onPartitionFenced(topicPartition, requestEpoch)
         true
```

Comment:
Shouldn't we return the result of `onPartitionFenced`?

Reply:
you are right again :)